inc/options/cdn.php

```diff
@@ -365,13 +769,41 @@
 			<tr>
 				<th colspan="2">
 					<ul>
-						<li><?php _e( 'You can use placeholders {wp_content_dir}, {plugins_dir}, {uploads_dir} instead of writing folder paths (wp-content, wp-content/plugins, wp-content/uploads).', 'w3-total-cache' ) ?></li>
-						<li><?php _e( 'If using Amazon Web Services or Self-Hosted <acronym title="Content Delivery Network">CDN</acronym> types, enable <acronym title="Hypertext Transfer Protocol">HTTP</acronym> compression in the "Media &amp; Other Files" section on <a href="admin.php?page=w3tc_browsercache">Browser Cache</a> Settings tab.', 'w3-total-cache' ) ?></li>
+						<li><?php esc_html_e( 'You can use placeholders {wp_content_dir}, {plugins_dir}, {uploads_dir} instead of writing folder paths (wp-content, wp-content/plugins, wp-content/uploads).', 'w3-total-cache' ); ?></li>
+						<li>
+							<?php
+							echo wp_kses(
+								sprintf(
+									// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag,
+									// translators: 3 opening HTML acronym tag, 4 closing HTML acronym tag,
+									// translators: 5 opening HTML a tag to W3TC BrowserCache admin page, 6 closing HTML a tag.
+									__(
+										'If using Amazon Web Services or Self-Hosted %1$sCDN%2$s types, enable %3$sHTTP%4$s compression in the "Media &amp; Other Files" section on %5$sBrowser Cache%6$s Settings tab.',
+										'w3-total-cache'
+									),
+									'<acronym title="' . __( 'Content Delivery Network', 'w3-total-cache' ) . '">',
```

Comment:
Use `esc_attr__()` for the 2 lines.

Reply:
Switched to esc_attr__ for 2 lines and changed link to use esc_url/admin_url functions